src/game_database.py

```diff
@@ -0,0 +1,179 @@
+        conn_type = config.connection_type
+        user = config.user
+        password = config.password
+        host = config.host
+        port = config.port
+        database = os.environ.get("DATABASE_NAME")
+        engine_string = "{}://{}:{}@{}:{}/{}".format(conn_type, user, password, host, port, database)
+
+    #create engine
+    engine = sql.create_engine(engine_string)
+
+    Base.metadata.create_all(engine)
+
+
+    # make session
+    Session = sessionmaker(bind=engine)
+    session = Session()
+    # Delete anything that's already in the tables
+    try:
+        session.execute(text("DELETE FROM metacritic"))
+        session.execute(text("DELETE FROM sales"))
+        session.execute(text("DELETE FROM steam"))
+        session.execute(text("DELETE FROM customer"))
+    except:
+        pass
+
+    #define variables
+    CLEAN_FILE_LIST=config.CLEAN_DATA_FILE_PATHS
+    cleaned_metacritic = pd.read_csv(CLEAN_FILE_LIST[0]).set_index('id')
+    cleaned_sales = pd.read_csv(CLEAN_FILE_LIST[1]).set_index('id')
+    cleaned_steam = pd.read_csv(CLEAN_FILE_LIST[2]).set_index('id')
+
+    # iterate through rows, add and commit for all 4 tables
+    metacritic_list=[]
+    for idx,features in tqdm(cleaned_metacritic.iterrows()):
+            metacritic_sample=Metacritic(id=int(idx),Game=str(features['Game']), Genre=str(features['Genre']),Metascore=float(features['metascore']),
+                                    Userscore=float(features['userscore']))
+            metacritic_list.append(metacritic_sample)
+
+    session.add_all(metacritic_list)
+
+    session.commit()
+    logging.info("Database created with Metacritic scored games table")
+
+    sales_list=[]
+    for idx,features in tqdm(cleaned_sales.iterrows()):
+
+            sales_sample=Sales(id=int(idx),Game=str(features['Game']),Genre=str(features['Genre']),NA_sales=float(features['NA_Sales']),EU_sales=float(features['EU_Sales']),
+                           JP_sales=float(features['JP_Sales']),Other_sales=float(features['Other_Sales']),Global_sales=float(features['Global_Sales']))
+            sales_list.append(sales_sample)
+    session.add_all(sales_list)
+
+    session.commit()
+    logging.info("Database created with Sales of games table")
+
+    steam_list=[]
+    for idx,features in tqdm(cleaned_steam.iterrows()):
+        steam_sample = Steam(id=int(idx),Userid=str(features['userid']),Game=str(features['Game']),Playtime=str(features['playtime']))
+        steam_list.append(steam_sample)
+    session.add_all(steam_list)
+
+    session.commit()
+    logging.info("Database created with Steam PC games table")
+
+    customer_sample=Customer(id=1,customer_first_name='John',customer_last_name='Doe',
+                             customer_fav_game='FIFA 19',customer_fav_genre='Sports',customer_time_spent=5.0,customer_rating=8)
+
+    session.add(customer_sample)
+    session.commit()
+    logging.info("Database created with Steam PC games table")
+
+    session.close()
```

Comment:
this is a humongous function. would highly recommend breaking it down.

Reply:
Yes there is a way of doing that. Thanks for pointing out!